VTUL/dspace-aip-archiver
# Harvest new & changed items from DSpace, package in APTrust bagit files, and upload them to a S3 bucket

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1676) (:star:)
https://webapps.es.vt.edu/jira/browse/LIBTD-2182
https://webapps.es.vt.edu/jira/browse/LIBTD-2184
https://webapps.es.vt.edu/jira/browse/LIBTD-2186

# What does this Pull Request do? (:star:)
This program fetches items that have been created or changed in
defined time period from DSpace using OAI-PMH protocol, creates an APTrust Bagit file for each exported Archival Information Package (AIP), and uploads files to the designated S3 bucket.

# What's the changes? (:star:)
Please see the updated README.md

# How should this be tested?
1. Deploy this program to a dev server with DSpace installed. 
2. Follow the readme.md and update the config file
3. Run the program
4. Verify output

# Interested parties
Tag (@jimtuttle @soumikgh @VTUL/dld-dev ) interested parties

(:star:) Required fields
